Source/Core/RequestScheduler.js

```diff
@@ -129,41 +256,93 @@ define([
      * }
      *
      */
-    RequestScheduler.throttleRequest = function(url, requestFunction) {
+    RequestScheduler.throttleRequest = function(request) {
```

Comment:
I know this was the existing name, but the most common function should have the most common name.  Perhaps this should be `request`, and the current `request` should be `deferRequest`.

I'm also not 100% sold on the "defer" language throughout, perhaps it maps well to promises, in which case it is fine; otherwise, I feel like `requestThisFrame` and `requestAnyFrame` (or something like that) is better.